db/structure.sql

```diff
@@ -1930,6 +1930,7 @@ INSERT INTO `schema_migrations` (version) VALUES
 ('20181213153332'),
 ('20190205200912'),
 ('20190920181828'),
-('20191113214448');
+('20191113214448'),
+('20191114215421');
```

Comment:
What is the additional migration id? There are no db changes in this PR, so there shouldn't be any changes in `structure.sql`.